rotkehlchen/chain/balances.py

```diff
@@ -46,7 +47,7 @@ class BlockchainBalances:
     avax: defaultdict[ChecksumEvmAddress, BalanceSheet] = field(init=False)
 
     @overload
-    def get(self, chain: SUPPORTED_EVM_CHAINS) -> defaultdict[ChecksumEvmAddress, BalanceSheet]:
+    def get(self, chain: Union[SUPPORTED_EVM_CHAINS, CHAINS_WITH_PROTOCOLS_WITH_BALANCE]) -> defaultdict[ChecksumEvmAddress, BalanceSheet]:  # noqa: E501
```

Comment:
The type here is wrong. `SUPPORTED_EVM_CHAINS` is a superset of `CHAINS_WITH_PROTOCOLS_WITH_BALANCE`, so it's redundant.

If it does not work with mypy then you either have a bug somewhere else, or there is a mypy bug and you should use type: ignore and explain why.